getter/main.go

```diff
@@ -0,0 +1,48 @@
+package getter
```

Comment:
This must be `locodedb`. This is the main package we want to export here. All the other code can be `internal/...` to `cmd`.